CorsixTH/Luatest/TH.lua

```diff
@@ -40,6 +40,10 @@ TheApp = {
     isCurrentSpeed = function(self, s) return s == self.speed end,
     gameLog = function() end,
   },
+  animation_manager = {
+    setPatientMarker = function(...) end,
+    setStaffMarker = function(...) end,
+  },
```

Comment:
I see no issue here myself adding a dummy function; unless it's meant to be actually testing some animation?

Reply:
My goal was to fix the false positive error. It was not to write tests whether you can set a marker.